Src/DependencyCollector/Shared/Implementation/ClientServerDependencyTracker.cs

```diff
@@ -44,8 +44,6 @@ internal static DependencyTelemetry BeginTracking(TelemetryClient telemetryClien
 
             activity.Stop();
 #else
-            telemetryClient.Initialize(telemetry);
-
             // telemetry is initialized by Base SDK OperationCorrealtionTelemetryInitializer
```

Comment:
it is still executed few lines of code above, I just removed second call to Initialize